Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionView.swift

```diff
@@ -167,9 +167,11 @@ public struct FaceLivenessDetectorView: View {
             .onReceive(viewModel.$livenessState) { output in
                 switch output.state {
                 case .completed:
+                    viewModel.livenessService.closeSocket(with: .normalClosure)
```

Comment:
Following up on thread from slack for context, on web after sending the empty video chunk to signal the end of video frames, the liveness service sends back a disconnect event and closes the connection for us. 

Not sure if that would apply here to the `.completed` state but I would wonder if the connection is already closed at this point and would that affect this at all?

Reply:
Thanks. This would be a no-op if the websocket connection has already completed. But after discussion with the service team, we shouldn't be sending a close frame on successful completion. Addressed in [b18118e](https://github.com/aws-amplify/amplify-ui-swift-liveness/pull/39/commits/b18118e45470950e1fd1a802f750993a4a39b4fd).